fixtures/dom/src/components/fixtures/number-inputs/NumberTestCase.js

```diff
@@ -5,7 +5,7 @@ const React = window.React;
 class NumberTestCase extends React.Component {
   state = {value: ''};
   onChange = event => {
-    const parsed = parseFloat(event.target.value, 10);
+    const parsed = parseFloat(event.currentTarget.value, 10);
```

Comment:
event.target was always the window in IE9. That seems like a bug too. I'm planning to investigate.

Reply:
Filed here: https://github.com/facebook/react/issues/12506. Pretty strange, I can't reproduce this outside of the test fixtures.